Android/3D Cave Runner - Android Studio/app/proguard-rules.pro

```diff
@@ -0,0 +1,173 @@
+-dontwarn **
+
+-keep class bitter.jnibridge.** { *; }
+-keep class com.unity3d.** { *; }
+-keep class org.fmod.** { *; }
+
+
+-keep public class com.google.android.gms.* {
+    public *;
+}
+-dontwarn com.google.android.gms.**
+
+
+-keep public class com.skillzgames.** { *; }
+-keep public class com.facebook.react.** { *; }
+-keep public class com.skillz.** { *; }
+-keep public class com.skillz.Skillz {
+    *;
+}
```

Comment:
Doesn't `com.skillz.** ` cover  `com.skillz.Skillz` ? Can you try removing `com.skillz.Skillz?`